WordPress/gutenberg
# Packages: Ensure that private packages do not update when publishing to npm

## Description


Changes included ensure that private packages are never updated during npm publishing. We tested [`--no-private`](https://github.com/lerna/lerna/tree/main/core/filter-options#--no-private) flag when publishing npm packages with changes for WordPress 5.9.1 release last Thursday and it worked as documented in the Lerna repository. I also included a command that always picks `patch` version updates for `npm run publish:patch` which simplifies the process.

## Testing Instructions
I executed the command `npm run publish:patch` to ensure that all non-private packages get automatic patch version updated. I bailed out early to avoid the unnecessary npm publishing.

## Screenshots 

<img width="882" alt="Screenshot 2022-02-21 at 10 59 57" src="https://user-images.githubusercontent.com/699132/154938052-3e7f8cb6-3197-42b0-91af-fb8310acdefb.png">


## Types of changes
Enhancement.

## Checklist:
